docs/Tutorials/Contracts/Deploying-Contracts.md

```diff
@@ -134,16 +132,21 @@ The Developer Quickstart provides an [example of a public transaction script](ht
     so we append that to the binary value).
```

Comment:
in this section I would change
* `from` - address of the EthSigner account. For example `0xfe3b557e8fb62b89f4916b721be55ceb828dbd73`.
to
* `from` - address of the sending account. For example `0xfe3b557e8fb62b89f4916b721be55ceb828dbd73`.